engine/src/main/java/org/terasology/math/ChunkMath.java

```diff
@@ -575,67 +574,67 @@ public static BlockRegion getEdgeRegion(BlockRegion region, Side side, BlockRegi
+                    region.minZ()
                 ).setMax(
-                    region.getMaxX(),
-                    region.getMinY(),
-                    region.getMaxZ());
+                    region.maxX(),
+                    region.minY(),
+                    region.maxZ());
             case LEFT:
                 return dest.setMin(
-                    region.getMinX(),
-                    region.getMinY(),
-                    region.getMinZ()
+                    region.minX(),
+                    region.minY(),
+                    region.minZ()
                 ).setMax(
-                    region.getMinX(),
-                    region.getMaxY(),
-                    region.getMaxZ());
+                    region.minX(),
+                    region.maxY(),
+                    region.maxZ());
             case RIGHT:
                 return dest.setMin(
-                    region.getMaxX(),
-                    region.getMinY(),
-                    region.getMinZ()
+                    region.maxX(),
+                    region.minY(),
+                    region.minZ()
                 ).setMax(
-                    region.getMaxX(),
-                    region.getMaxY(),
-                    region.getMaxZ());
+                    region.maxX(),
+                    region.maxY(),
+                    region.maxZ());
             case FRONT:
                 return dest.setMin(
-                    region.getMinX(),
-                    region.getMinY(),
-                    region.getMinZ()
+                    region.minX(),
+                    region.minY(),
+                    region.minZ()
                 ).setMax(
-                    region.getMaxX(),
-                    region.getMaxY(),
-                    region.getMinZ());
+                    region.maxX(),
+                    region.maxY(),
+                    region.minZ());
             case BACK:
                 return dest.setMin(
-                    region.getMinX(),
-                    region.getMinY(),
-                    region.getMaxZ()
+                    region.minX(),
+                    region.minY(),
+                    region.maxZ()
                 ).setMax(
-                    region.getMaxX(),
-                    region.getMaxY(),
-                    region.getMaxZ());
+                    region.maxX(),
+                    region.maxY(),
+                    region.maxZ());
             default:
                 return dest.setMin(
-                    region.getMinX(),
-                    region.getMinY(),
-                    region.getMinZ()
+                    region.minX(),
+                    region.minY(),
+                    region.minZ()
                 ).setMax(
-                    region.getMaxX(),
-                    region.getMaxY(),
-                    region.getMaxZ()
+                    region.maxX(),
+                    region.maxY(),
+                    region.maxZ()
                 );
```

Comment:
for these you need to use `set` the dest can become invalid between setMin and setMax

Reply:
Done 👍